skunk/rust/skunk/src/ast.rs

```diff
@@ -468,6 +468,7 @@ pub enum GraphDirective {
 #[derive(Debug, PartialEq)]
 pub struct Use {
   pub name: String,
```

Comment:
Nit: would be good to see more consistent formatting.

I normally just use `cargo fmt` as it's pretty good and is *mostly* more readable. YMMV

Reply:
I am not generally OK with autoformat, or with fixed line lengths. I realize this flies counter to established wisdom.

Reply:
In this case, it was the lack of consistent indentation (or indentation at all) rather than line lengths that I was trying to point out.

I know of your stance and empathise.

Nice PR, cheers!